static/app/views/releases/list/index.tsx

```diff
@@ -551,6 +547,7 @@ class ReleasesList extends DeprecatedAsyncView<Props, State> {
 
     return (
       <PageFiltersContainer showAbsolute={false}>
+        <SentryDocumentTitle title={t('Releases')} />
```

Comment:
```suggestion
        <SentryDocumentTitle title={t('Releases')} orgSlug={organization.slug} />
```
does this one need `orgSlug` too?